mem.go

```diff
@@ -23,22 +24,53 @@ func (ca *Cagent) MemResults() (MeasurementsMap, error) {
 
 	memStat, err := mem.VirtualMemoryWithContext(ctx)
 
+	results = map[string]interface{}{
+		"total_B":           nil,
+		"free_B":            nil,
+		"free_percent":      nil,
+		"cached_B":          nil,
+		"cached_percent":    nil,
+		"shared_B":          nil,
+		"shared_percent":    nil,
+		"buff_B":            nil,
+		"buff_percent":      nil,
+		"used_B":            nil,
+		"used_percent":      nil,
+		"available_B":       nil,
+		"available_percent": nil,
+	}
+
 	if err != nil {
```

Comment:
Please fix this `if` branch.
Return early in error case and then you can omit the whole `else` branch.
```suggestion
	if err != nil {
		log.Errorf("[MEM] Failed to get virtual memory stat: %s", err.Error())
		return nil, errors.New("MEM: " + err.Error())
	}
	results["total_B"] = memStat.Total
	...
```

Reply:
fixed